eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/CClientDemo.java

```diff
@@ -49,11 +51,10 @@ public static void main(String[] args) throws Exception {
         client.registerSubBusiHandler(new ReceiveMsgHook() {
             @Override
             public void handle(Package msg, ChannelHandlerContext ctx) {
-                if (msg.getHeader().getCmd() == Command.ASYNC_MESSAGE_TO_CLIENT
-                        || msg.getHeader().getCmd() == Command.BROADCAST_MESSAGE_TO_CLIENT) {
-
+                if (Objects.equals(msg.getHeader().getCmd(), Command.ASYNC_MESSAGE_TO_CLIENT)
```

Comment:
Therefore we can conclude that when we use == operator for enum member comparison then it performs type compatibility check at compile time whereas .equals() method will never worry about the types of both the arguments.So suggest use == operator.